### ISSUE TITLE
Clicking on a to-do and calendar links to a blank screen

### ISSUE THREAD

**Issue by lucastbelem** [jesus-collective/mobile#760]
There is a course in dev called "February 26th course". I created 2 test assignments in there. They appear in the To-Do list, but when I click on either, assignments do not display.

**Comment by JonEsparaz** [comment]
Are you sure they appear in the to-do list?

![image](https://user-images.githubusercontent.com/48423418/109333922-099c5080-782e-11eb-8788-d4d453676d45.png)

**Comment by JonEsparaz** [comment]
Never mind, the due dates had passed already 🤦🏻‍♂️

**Comment by JonEsparaz** [comment]
```typescript diff
  goToLesson(actions: CourseActions, week: string, lesson: string): void {
    actions.setActiveWeek(parseInt(week, 10) - 1) // undo fix for off-by-one-error here
    actions.setActiveLesson(parseInt(lesson, 10) - 1)
    actions.setActiveScreen("Details")
  }
```

@lucastbelem, I've fixed the issue, but...

At some point there was an [off-by-one-error](https://en.wikipedia.org/wiki/Off-by-one_error) where course weeks were 0-indexed instead of 1-indexed, so I adjusted the code to handle the erroneous data. Now it appears that courses are correctly 1-indexed. The problem: clicking the to-do list and calendar in old courses (created when the off-by-one-error still existed) will not work going forward. 

If you are only creating new courses in the beta environment (and we delete all the erroneous courses in dev), then there shouldn't really be a problem.

TL;DR: there's an easy fix for "new" courses, but the any "old" course's to-do list and calendar will have a similar issue.

**Comment by lucastbelem** [comment]
@JonEsparaz thanks man. Technically, any course that exists on dev or beta are test courses, so we're good there.
